app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2428,12 +2428,12 @@ public void onClick(DialogInterface dialog, int which) {
                         }
                     };
                 mProgressDialog.setIcon(android.R.drawable.ic_dialog_info);
-                mProgressDialog.setTitle(StringUtils.getStringRobust(this, R.string.saving_form));
-                        mProgressDialog.setMessage(StringUtils.getStringRobust(this, R.string.please_wait));
+                mProgressDialog.setTitle(StringUtils.getStringSpannableRobust(this, R.string.saving_form));
+                        mProgressDialog.setMessage(StringUtils.getStringSpannableRobust(this, R.string.please_wait));
                                 mProgressDialog.setIndeterminate(true);
                 mProgressDialog.setCancelable(false);
-                mProgressDialog.setButton(StringUtils.getStringRobust(this, R.string.cancel), savingButtonListener);
-                        mProgressDialog.setButton(StringUtils.getStringRobust(this, R.string.cancel_saving_form),
+                mProgressDialog.setButton(StringUtils.getStringSpannableRobust(this, R.string.cancel), savingButtonListener);
+                        mProgressDialog.setButton(StringUtils.getStringSpannableRobust(this, R.string.cancel_saving_form),
```

Comment:
Do we need the word "Spannable" in here? Are we using getStringRobust still anywhere?